controllers/vaultstaticsecret_controller.go

```diff
@@ -6,27 +6,37 @@ package controllers
 import (
 	"context"
 	"encoding/base64"
+	"encoding/json"
 	"fmt"
+	"strings"
 	"time"
 
 	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/tools/record"
+	"nhooyr.io/websocket"
```

Comment:
Out of curiosity, is this the same module that we are using in agent?

Reply:
Yeah, it's used in agent and vault's CLI: https://github.com/search?q=org%3Ahashicorp+%22nhooyr.io%2Fwebsocket%22+repo%3Ahashicorp%2Fvault&type=code